virt/virtual_machines/virtual_disks/virt-vm-disk-enlargment.adoc

```diff
@@ -0,0 +1,16 @@
+[id="virt-vm-disk-enlargment"]
+= Virtual machine disk enlargement
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-vm-disk-enlargment
+
+toc::[]
+
+You can enlarge the size of a virtual machine's (VM) disk to provide a greater storage capacity.
+
+include::modules/virt-enlarging-vm-disk.adoc[leveloffset=+1]
+
+== Additional resources
```

Comment:
This needs a unique ID above it. Something like `[id="additional-resources_virt-vm-disk-enlargement"]`